### PR TITLE
Kotlin 1.9.10 -> 1.9.24

### PR DESCRIPTION
**Pull request by cortinico**
## Summary

This bumps Kotlin from 1.9.10 to 1.9.24 making it easier to bump to 2.0.0

## How It Was Tested

See app running in screeshot below:

## Screenshot/Gif

<details>

<summary>App running with Kotlin 1.9.24</summary>

screen-1720094095

</details>

### CODE REVIEW

File: zeapp/build.gradle.kts [hunk 1]
```diff
@@ -9,7 +9,7 @@ plugins {
     alias(libs.plugins.license.report.gradle) apply false
     alias(libs.plugins.aboutlibraries.gradle) apply false
     id("com.android.test") version "8.0.2" apply false
-    kotlin("multiplatform") version "1.9.10" apply false
+    kotlin("multiplatform") version "1.9.24" apply false
```
Comment (alisen) [inline zeapp/build.gradle.kts:12]: Could you use toml file here as well?

Comment (cortinico) [reply zeapp/build.gradle.kts]: Done here be3df03

**Comment by alisen** [comment]
Please, resolve the conflicts @cortinico

**Comment by cortinico** [comment]
@alisen this should now be ready to go 👍

**Comment by alisen** [comment]
> @alisen this should now be ready to go 👍

All checks are failing. Could you try to fix it?

**Comment by cortinico** [comment]
> > @alisen this should now be ready to go 👍
> 
> All checks are failing. Could you try to fix it?

Yup my bad

**Review by alisen — APPROVED** [review]

**Pull request merged**